features/test_runner/fastlane_test_runner.rb

```diff
@@ -4,13 +4,35 @@ module FastlaneCI
   # Represents the test runner responsible for loading and running
   # fastlane Fastfile configurations
   class FastlaneTestRunner < TestRunner
-    def run(platform: nil, lane: nil, parameters: nil)
+    # Parameters for running fastlane
+    attr_reader :platform
+    attr_reader :lane
+    attr_reader :parameters
+
+    def initialize(platform: nil, lane: nil, parameters: nil)
+      @platform = platform
+      @lane = lane
+      @parameters = parameters
+    end
+
+    def run
+      raise "No block provided for `run` method" unless block_given?
       require "fastlane"
 
       ci_output = FastlaneCI::FastlaneCIOutput.new(
         file_path: "fastlane.log",
-        block: proc do |row|
-          puts "Current output from fastlane: #{row}"
+        each_line_block: proc do |row|
+          # Additionally to transfering the original metadata of this message
+          # that look like this:
+          #
+          # {:type=>:success, :message=>"Everything worked"}
+          #
+          # we append the HTML code that should be used in the `html` key
+          # the result looks like this
+          #
+          # {"type":"success","message":"Driving the lane 'ios beta'","html":"<p class=\"success\">Driving the lane 'ios beta'</p>"}
+          #
+          row[:html] = FastlaneOutputToHtml.convert_row(row)
           yield(row)
```

Comment:
This was failing for me (on `master`) `yield` misses a callable, where is it? 😕

Reply:
Sorry, my bad that it was failing on master, it's over here now https://github.com/fastlane/ci/blob/e1edb77c5904036f8b8ba6280fadc04dd0df9242/services/test_runner_service.rb#L108-L110